content/millennium/r4/foundation/security/provenance.md

```diff
@@ -42,7 +42,7 @@ Search for Provenances that meet supplied query parameters:
  Name         | Required? | Type          | Description
 --------------|-----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
  `_id`        | This or `target` | [`token`]     | The logical resource id associated with the resource. Example: `doc-7845`
- `target`     | This or `_id` | [`reference`] | Target Reference(s) (usually version specific). The AllergyIntolerance, Condition, Device, DocumentReference, Encounter, Immunization, MedicationRequest, Observation, Patient and Procedure resources are supported. Example: `DocumentReference/66547`
+ `target`     | This or `_id` | [`reference`] | Target Reference(s) (usually version specific). The AllergyIntolerance, Condition, DocumentReference, Encounter, Immunization, MedicationRequest, Observation, Patient and Procedure resources are supported. Example: `DocumentReference/66547`
```

Comment:
I guess for Patient also we don't have the tables, so we need to remove it from here